server/routes.go

```diff
@@ -165,5 +171,12 @@ func getRoutes(app *middleware.App) []Route {
 			middleware.WrapApp(app, controllers.FetchAllProjectStats),
 			false,
 		},
+		{
+			"Accept/Reject Project",
+			"POST",
+			"/project/updt_status",
```

Comment:
I think `/project/update` is better.

Reply:
In the future it can be used to change other details too.

Reply:
There is a `/project/update` endpoint

Reply:
Updated `/project/update` to allow approval for organisers